### ISSUE TITLE
timer hud and simple chat processor problem

### ISSUE THREAD

**Issue by gasolino** [0blu/Timer#8]
Hi, sometimes, when my server restart, the hud don't work.. error logs:
L 01/19/2016 - 19:06:01: [SM] Native "Timer_GetPoints" reported: Plugin owning this native is currently paused.
L 01/19/2016 - 19:06:01: [SM] Displaying call stack trace for plugin "timer-hud_css.smx":
L 01/19/2016 - 19:06:01: [SM]   [0]  Line 1429, C:\Users\giogi\Desktop\sourcemod-1.7.3-git5290-windows\addons\sourcemod\scripting\timer-hud_css.sp::UpdateHUD_CSS()
L 01/19/2016 - 19:06:01: [SM]   [1]  Line 1287, C:\Users\giogi\Desktop\sourcemod-1.7.3-git5290-windows\addons\sourcemod\scripting\timer-hud_css.sp::HUDTimer_CSS()

and the simple chat processor don't work.. error logs:
[SM] Native "SetFailState" reported: Translation file is not present
L 01/19/2016 - 19:06:36: [SM] Displaying call stack trace for plugin "simple-chatprocessor.smx":
L 01/19/2016 - 19:06:36: [SM]   [0]  Line 185, C:\Users\giogi\Desktop\sourcemod-1.7.3-git5290-windows\addons\sourcemod\scripting\simple-chatprocessor.sp::OnPluginStart()

anyone can help me pls????
Bye

**Comment by sneak-it** [comment]
"[SM] Native "SetFailState" reported: Translation file is not present"

You didn't upload the SCP translation files.

**Comment by 0blu** [comment]
Yes, it seems to be a Problem with the traslation file, please Check this.
If its still not working, please list all Plugins in your plugins/ folder here.

**Comment by gasolino** [comment]
ok tnx, now I try

**Comment by gasolino** [comment]
the lists of plugins: 
] sm plugins
 "Admin Help" (1.7.3-dev+5273) by AlliedModders LLC
 "[Timer] Teams" (2.5.0.0) by Zipcore, Jason Bourne
 "MapChooser" (1.7.3-dev+5273) by AlliedModders LLC
 "Reserved Slots" (1.7.3-dev+5273) by AlliedModders LLC
 "[Timer] Random Startmap" (2.5.0.0) by Zipcore
 "[Timer] Sounds" (2.5.0.0) by Zipcore, Jason Bourne
 "Player Commands" (1.7.3-dev+5273) by AlliedModders LLC
 "MapChooser Extended" (1.10.2) by Powerlord, Zuko, and AlliedModders LLC
 "[Timer] Rankings" (2.5.0.0) by Panduh (AlliedMods: thetwistedpanda), Zipcore
 "Rock The Vote" (1.7.3-dev+5273) by AlliedModders LLC
To see more, type "sm plugins 11"
] sm plugins 11
 "[Timer] Physicsinfo" (2.5.0.0) by Zipcore
 "[Timer] Logging" (2.5.0.0) by Zipcore, Credits: Alongub
 "Fun Commands" (1.7.3-dev+5273) by AlliedModders LLC
 "[Timer] World Record" (2.5.0.0) by Zipcore, Credits: Alongub
 "[Timer] NoJail" (1.0) by Smesh; Credit: Zipcore
 "[Timer] HUD" (2.5.0.0) by Zipcore, Alongub
 "[Timer] MapZones" (2.5.0.0) by Zipcore, Credits: Alongub
 "SurfTimer TempInfos" (1.0) by das d!
 "[Timer] Players Online DB" (1.0) by Zipcore
 "[Timer] Autostrafe" (1.0) by Zipcore, Credits: Mev
 "[Timer] Start zone no jump" (2.5.0.0) by Zipcore, Rop
To see more, type "sm plugins 22"
] sm plugins 22
 "Fun Votes" (1.7.3-dev+5273) by AlliedModders LLC
 "[Timer] Spectate" (2.5.0.0) by Zipcore, Jason Bourne
 "[Timer] cpMod" (2.5.0.0) by Zipcore, byaaaaah
 "[Timer] Map Tier System" (2.5.0.0) by Zipcore
 "Basic Comm Control" (1.7.3-dev+5273) by AlliedModders LLC
 "[Timer] Quickstyle Commands" (2.5.0.0) by Zipcore
 "[Timer] MapInfo" (2.5.0.0) by Zipcore
 "[Timer] Physics" (2.5.0.0) by Zipcore, Alongub
 "Anti-Flood" (1.7.3-dev+5273) by AlliedModders LLC
 "SurfTimer TempPlayerinfo" (1.0) by das d!
 "Nextmap" (1.7.3-dev+5273) by AlliedModders LLC
To see more, type "sm plugins 33"
] sm plugins 33
 "Auto Respawn" (1.3.3.9) by das d!
 "Sound Commands" (1.7.3-dev+5273) by AlliedModders LLC
 "[Timer] Rankings - Geo-ranking" (2.5.0.0) by Zipcore
 "[Timer] MapZones - Damage Controller" (2.5.0.0) by Zipcore, Credits: Alongub
 "[Timer] Finish Manager" (2.5.0.0) by Zipcore
 "Client Preferences" (1.7.3-dev+5273) by AlliedModders LLC
 "Flashboost Fix" (1.1) by mev, zipcore, m_bNightstalker
 "[Timer] Rankings - Top10 Extend" (1.0) by Zipcore
 "[Timer] Hide Commands" (2.5.0.0) by Rop
 "Admin File Reader" (1.7.3-dev+5273) by AlliedModders LLC
 "[Timer] Autospawn" (2.5.0.0) by Zipcore, Credits: Das D
To see more, type "sm plugins 44"
] sm plugins 44
 "[Timer] Hide" (2.5.0.0) by Zipcore, exvel
 "RandomCycle" (1.7.3-dev+5273) by AlliedModders LLC
 "[Timer] Strafe Stats" (2.5.0.0) by Zipcore, Miu
 "[Timer] MySQL Manager" (2.5.0.0) by Zipcore
 "button_announcer.smx" button_announcer.smx
 "[Timer] Worldrecord - MapTop" (1.0) by Zipcore, Credits: Das D
 "[Timer] Core" (2.5.0.0) by Zipcore, Credits: Alongub
 "[Timer] Maplist Helper" (2.5.0.0) by Zipcore
 "Basic Votes" (1.7.3-dev+5273) by AlliedModders LLC
 "[Timer] Worldrecord - PlayerInfo" (1.0) by Zipcore, Credits: Das D
 "Basic Commands" (1.7.3-dev+5273) by AlliedModders LLC
To see more, type "sm plugins 55"
] sm plugins 55
 "Map Nominations" (1.7.3-dev+5273) by AlliedModders LLC
 "Basic Ban Commands" (1.7.3-dev+5273) by AlliedModders LLC
 "Admin Menu" (1.7.3-dev+5273) by AlliedModders LLC
 "[Timer] Worldrecord - Latest WRs" (1.0) by Zipcore
 "[Timer] Auto lower case commands" (2.5.0.0) by Zipcore
 "Trikz (Redux)" (1.7.3) by shavit
 "[Timer] Main Menu" (2.5.0.0) by Zipcore
 "Simple Chat Processor (Redux)" (2.0.2 (modified by Zipcore)) by Simple Plugins, Mini
 "[Timer] MapZones - Simple Stage Timer" (1.0) by Zipcore
 "[Timer] TeleMe" (2.5.0.0) by Zipcore
 "Basic Info Triggers" (1.7.3-dev+5273) by AlliedModders LLC
To see more, type "sm plugins 66"
] sm plugins 66
 "[Timer] Challenge Points Lite" (2.5.0.0) by Zipcore
 "Basic Chat" (1.7.3-dev+5273) by AlliedModders LLC
 "[Timer] Finish Message" (1.0) by Zipcore
 "[Timer] Rankings - Points Lite" (2.5.0.0) by Zipcore
 "[Timer] LJstats" (2.5.0.0) by justshoot, Zipcore
 "Simple Chat Colors" (2.0.0) by Simple Plugins
 "SM Weapon Cleanup" (1.2) by TechKnow

Idk ,don't work =( help me plss

**Comment by gasolino** [comment]
![immagine](https://cloud.githubusercontent.com/assets/11800974/12457074/ece6dd50-bfa2-11e5-94f9-90d74088cef4.png)

**Comment by sneak-it** [comment]
I already told you how to fix your problem, why do you ignore me? The error CLEARLY states what your issue is.

Your error: "[SM] Native "SetFailState" reported: Translation file is not present"
Emphasis on: "Translation file is not present"

You didn't upload the SCP translation files, so upload the translation files to the server! They are located under /addons/sourcemod/translations/

**Comment by gasolino** [comment]
I have put the translation...

**Comment by gasolino** [comment]
this is the problem

**Comment by sneak-it** [comment]
The error is telling you that the translation files are not present. Clearly you have made a mistake. I can't make up the error that you provided.

**Comment by gasolino** [comment]
ok, in the logs, there is no more this problem "[SM] Native "SetFailState" reported: Translation file is not present".. It does not work the same

**Comment by gasolino** [comment]
sry, I'm not good
